arches_for_science/models.py

```diff
@@ -17,4 +21,40 @@ class Meta:
     def __init__(self, *args, **kwargs):
         super(RendererConfig, self).__init__(*args, **kwargs)
         if not self.configid:
-            self.configid = uuid.uuid4()
\ No newline at end of file
+            self.configid = uuid.uuid4()
+
+
+class ManifestXDigitalResource(models.Model):
+    manifest = models.TextField(unique=True)
+    digitalresource = models.TextField(unique=True)
+    class Meta:
+        managed = True
+        db_table = "manifest_x_digitalresource"
+
+class CanvasXDigitalResource(models.Model):
+    canvas = models.TextField(unique=True)
+    digitalresource = models.TextField(unique=True)
+
+    class Meta:
+        managed = True
+        db_table = "canvas_x_digitalresource"
+
+class ManifestXCanvas(models.Model):
+    manifest = models.TextField()
+    canvas = models.TextField()
+
+    class Meta:
+        managed = True
+        db_table = "manifest_x_canvas"
+
+
+@receiver(post_save, sender=IIIFManifest)
+def create_digital_resources(sender, instance, created, **kwargs):
+    from arches_for_science.utils.digital_resource_for_manifest import digital_resources_for_manifest, digital_resources_for_canvases
```

Comment:
Do these imports not work from the top of the file?

Reply:
The functions are importing the models (from this file). So, I didn't find a way to avoid the circular import.